eng/linkerTests/linkerCorrectnessTests.targets

```diff
@@ -0,0 +1,109 @@
+<Project>
+  <PropertyGroup>
+    <LinkerTestDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsBinDir)', 'linkerTests'))</LinkerTestDir>
+    <LinkerTestProjectsDir>$([MSBuild]::NormalizeDirectory('$(LinkerTestDir)', 'projects'))</LinkerTestProjectsDir>
+    <TestDotNetPath>$([MSBuild]::NormalizePath('$(DotNetRoot)', 'dotnet'))</TestDotNetPath>
```

Comment:
Aren't you running the linker on Windows? Don't you need .exe there? At least when you quote the path it should be required.

Reply:
Why not use `DotNetTool` instead? https://github.com/dotnet/arcade/blob/dcc5ca2cf397c53575d376e80ecd30f63160037f/src/Microsoft.DotNet.Arcade.Sdk/tools/RepoLayout.props#L35-L36

Reply:
you don't need it, in Windows when you call an application inside a folder without using the .exe, the default probing mechanism will find dotnet.exe for you.